src/main/kotlin/dev/stalla/model/itunes/ItunesCategory.kt

```diff
@@ -1,43 +1,348 @@
 package dev.stalla.model.itunes
 
-import dev.stalla.builder.ItunesCategoryBuilder
-import dev.stalla.builder.validating.ValidatingItunesCategoryBuilder
-import dev.stalla.model.BuilderFactory
+import dev.stalla.util.findItunesCategory
 
 /**
- * An [iTunes-style `<category>` tag][https://help.apple.com/itc/podcasts_connect/#/itcb54353390].
- * The same format is also used for Google Play <category> tags, just with a different namespace.
- *
- * @param name The name of the category.
+ * An [iTunes-style `<category>` tag][https://help.apple.com/itc/podcasts_connect/#/itc9267a2f12].
  */
-public sealed class ItunesCategory(public open val name: String) {
+public sealed interface ItunesCategory {
 
-    public companion object Factory : BuilderFactory<ItunesCategory, ItunesCategoryBuilder> {
+    /** The name of the category */
+    public val categoryName: String
+
+    public companion object Factory {
 
-        /** Returns a builder implementation for building [ItunesCategory] model instances. */
         @JvmStatic
-        override fun builder(): ItunesCategoryBuilder = ValidatingItunesCategoryBuilder()
-    }
+        public fun from(category: String?): ItunesCategory? = findItunesCategory(category)
+
+        @JvmField
+        public val ARTS: ItunesCategory = SimpleItunesCategory.ARTS
```

Comment:
The interfaces companion object is rather bloated, but otherwise we cannot provide a seamless Java API experience 😕. Same story for the sealed class approach.